File: JavaScript/1929.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

const firstLine = lines.shift().split(' ');

function ehTriangulo (a,b,c){
    return (Math.abs(b-c) < a && a < b+c) | (Math.abs(a-c) < b && b < a+c) | (Math.abs(a-b) < c && c < a+b) ? 1 : 0
}

var firstInput = parseInt(firstLine.shift());
var secondInput = parseInt(firstLine.shift());
var thirdInput = parseInt(firstLine.shift());
var fourthInput = parseInt(firstLine.shift());

if(ehTriangulo(firstInput,secondInput,thirdInput))
    console.log('S');
else if(ehTriangulo(firstInput,secondInput,fourthInput))
    console.log('S');
else if(ehTriangulo(firstInput,fourthInput,thirdInput))
    console.log('S');
else if(ehTriangulo(fourthInput,secondInput,thirdInput))
    console.log('S');
else
    console.log('N');

File: C/1929.c
#include <stdio.h>
#include <stdlib.h>
 
int main() {
 
  int ehTriangulo (int a, int b, int c){
      return (abs(b-c) < a && a < b+c) | (abs(a-c) < b && b < a+c) | (abs(a-b) < c && c < a+b);
  }

  int firstInput,secondInput,thirdInput,fourthInput;

  scanf("%d %d %d %d", &firstInput, &secondInput, &thirdInput, &fourthInput);

  if(ehTriangulo(firstInput,secondInput,thirdInput))
      printf("%c\n", 'S');
  else if(ehTriangulo(firstInput,secondInput,fourthInput))
      printf("%c\n", 'S');
  else if(ehTriangulo(firstInput,fourthInput,thirdInput))
      printf("%c\n", 'S');
  else if(ehTriangulo(fourthInput,secondInput,thirdInput))
      printf("%c\n", 'S');
  else
      printf("%c\n", 'N');
  return 0;
}
